modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -2504,6 +2510,16 @@ private boolean _isRequiredField(GraphQLError graphQLError) {
 				(String)path.get(path.size() - 1), "parent");
 		}
 
+		private boolean _isStatusException(GraphQLError graphQLError) {
```

Comment:
We don't want to include a dependency in Vulcan with Objects, as Vulcan is the REST API infrastucture, and in this case, we are only using the name of the class.

I have checked the Liferay code and I think we could consider a client exception whatever throwable class that ends with `StatusException`, so maybe we could consider that as the pattern to use in here.

Something like:

```java
Throwable throwable = _getThrowable(graphQLError);

if (StringUtil.endsWith(ClassUtil.getClassName(throwable), "StatusException")) {
	return true;
}

return false;
```

Reply:
I have checked and we are breaking this integration test locally:

```
com.liferay.portal.vulcan.internal.graphql.servlet.test.GraphQLServletTest > testQueryDepthLimit STARTED

com.liferay.portal.vulcan.internal.graphql.servlet.test.GraphQLServletTest > testQueryDepthLimit FAILED
```

Please, check it